app/utils.py

```diff
@@ -0,0 +1,16 @@
+from datetime import date, timedelta
+
+from app.scraper import menu, convert_daily_to_dict
+
+def get_upcoming_meals(days=7):
+    upcoming = []
+    for day in [date.today() + timedelta(d) for d in range(days)]:
+        upcoming.append(convert_daily_to_dict(menu[day]))
+    return upcoming
+
+def get_upcoming_meals_for_user(user, days=7):
+    meals_by_day = get_upcoming_meals(days)
+    preferences = [pref.food for pref in user.preferences]
+    upcoming = [(day['day'][:-2], meal, day[meal]) for day in meals_by_day 
+                for meal in day if day[meal] in preferences]
```

Comment:
1) We need to call `lower()` here as well.
2) We probably want to check to make sure all the words of the preference are in the meal, not the other way around.  For example, if it was "Beef Ragout with mashed potatoes" and their preference is "beef ragout" then that preference is a subset of the meal.